state_bucket/main.tf

```diff
@@ -75,6 +75,7 @@ resource "aws_s3_bucket" "s3-logs" {
 
   versioning {
     enabled = true
+    mfa_delete = var.env_name == "prod" ? true : false
```

Comment:
Let's pass this in as a var instead of having it trigger on a specific environment

Reply:
Made the changes and created a new varialbles.tf file in the sub-directory. Default is "false".